### PR TITLE
Add logo pg

### PR DESCRIPTION
**Pull request by JMurphyWeb**
#55 

@mk4111 @SimonLab tests are passing on one version but not the other. not sure why.

We need to update the heroku app and staging app to add the 2 new env variables before it will work. (see the google drive doc) I have already added them to travis.

I have opened a different PR relating to the styling.

@SimonLab I am not sure why we have 2 versions of node being run on travis. I think that is why one of them is failing.

### CODE REVIEW

**Comment by JMurphyWeb** [comment]
@mk4111 @SimonLab tests are passing on one version but not the other. not sure why.

We need to update the heroku app and staging app to add the new env variables before it will work. (see the google drive doc)

Regarding the styling i updated the description of this issue: https://github.com/emfoundation/ce100-app/issues/55#issue-171618189

**Comment by JMurphyWeb** [comment]
@SimonLab I am not sure why we have 2 versions of node being run on travis. I think that is why one of them is failing.

File: public/js/index.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+var logoSaver = (function () {
+  var file_name;
+
+  var saveFile = function (addedFile) {
+    console.log('saving file,', addedFile);
```
Comment (SimonLab) [inline public/js/index.js]: do we need this console.log?

File: public/js/index.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    xhttp.open('POST', '/orgs/' + id + '/edit', true);
+    console.log(id, formData);
+    xhttp.send(formData);
+  }
+
+  return {
+    saveFile: saveFile,
+    submitForm: submitForm
+  };
+})();
```
Comment (SimonLab) [inline public/js/index.js]: a few console.log to clean on this file I think

**Pull request merged**